xcp_d/utils/plot.py

```diff
@@ -512,9 +512,12 @@ def plot_svgx(rawdata,
 
     # Remove first N deleted from raw_data so it's same length as censored files
     if len(raw_dvars_data) > len(residual_dvars_data):
+        # TODO: Should this be [-len(residual_dvars_data):] ?
+        # ... Seems to grab first N, not last N.
```

Comment:
Yes, seems like this is the better way to implement it

Reply:
I've punted the issue to https://github.com/PennLINC/xcp_d/issues/589. This part of the bug is easy enough to fix, but I'd like to address it in a separate PR.